build/lib/extensions.js

```diff
@@ -232,6 +232,12 @@ const externalExtensions = [
     'sql-database-projects',
     'sql-migration'
 ];
+/**
+ * Extensions that are built into ADS but should be packaged externally as well for VS Code.
+ */
+exports.vscodeExternalExtensions = [
+    'data-workspace'
```

Comment:
only data workspace or projects too?

Reply:
This list is for extensions that are packaged into ADS that we also want to generate VS Code VSIXs for - since packaged extensions are normally skipped for this logic because this is only supposed to be for generating external VSIXs.

The sql proj extension is an externally packaged extension so doesn't needed to be added here - just having the package.vscode.json is enough in that case.